install/kubernetes/helm/istio/values.yaml

```diff
@@ -70,7 +70,6 @@ ingress:
 #
 sidecar-injector:
   enabled: false
-  serviceAccountName: default # used only if RBAC is not enabled
```

Comment:
Why removing service account here?

Reply:
My original intention is that if RBAC is enabled, chart will create its own service account, otherwise, use `default` service account. So no need to provide user with this option.
And I found the only code that uses the `serviceAccountName` in chart(https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/sidecar-injector/templates/configmap.yaml):
```
- name: istio-certs
        secret:
          optional: true
          {{ "{{ if eq .Spec.ServiceAccountName \"\" -}}" }}
          secretName: istio.default
          {{ "{{ else -}}" }}
          secretName: {{ "{{ printf \"istio.%s\" .Spec.ServiceAccountName }}"  }}
          {{ "{{ end -}}" }}
```

It seems the ServiceAccountName is duplicated and removing the line will not have any impact.

BTW, it's Okay for me to put it back.